cdc/entry/mounter.go

```diff
@@ -519,10 +526,13 @@ func getDefaultOrZeroValue(col *timodel.ColumnInfo) (interface{}, int, string, e
 			return emptyBytes, sizeOfEmptyBytes, "", nil
 		default:
 			d = table.GetZeroValue(col)
+			if d.IsNull() {
+				log.Error("meet unsupported column type", zap.String("column info", col.String()))
+			}
```

Comment:
What will happen in this scenario?

Reply:
sink null to the downstream and cause data inconsistency